lib/routers/establishment/project-versions.js

```diff
@@ -42,12 +43,16 @@ router.param('id', (req, res, next, id) => {
     .catch(next);
 });
 
-router.get('/:id', (req, res, next) => {
-  res.response = res.version;
-  next();
-});
+router.get('/:id',
+  permissions('project.read.single', (req, res) => ({ id: res.project.licenceHolderId })),
```

Comment:
I think this clearly slipped past me in one of the earlier PRs, but I'd probably prefer to be consistent with building params onto the request rather than the response. See https://github.com/UKHomeOffice/asl-public-api/blob/master/lib/routers/establishment/index.js#L21

Reply:
Also, having an `id` param on a project read permission _not_ be the project id seems super weird to me.

Reply:
I 100% agree with the `req` `res`, I thought I was being consistent by using `res` but I was obvs being consistent with the wrong thing! I'll do a sweep

Reply:
I also agree re the `id` param - I was just being lazy as this required _no_ code changes in permissions - but I'll do the right thing and add a `project:own` setting